antora-ui-camel/src/css/vars.css

```diff
@@ -6,6 +6,7 @@
   --color-smoke-50: #f5f5f5;
   --color-smoke-70: #f0f0f0;
   --color-smoke-90: #e1e1e1;
+  --color-smoke-100: #e3dcdc;
```

Comment:
Why introduce yet another color, could we not use `--color-smoke-90`?

Reply:
I was looking for a darker shade so that it is visible even in the brightest screens. I guess --colot-smoke-90 would work too.